packages/nativescript-google-login/common.ts

```diff
@@ -9,12 +10,16 @@ export abstract class Common extends NsObservable {
   constructor(config: GoogleLoginConfig, activity?: any) {
```

Comment:
Z tego co widzę to już to wymagasz, więc raczej nie może być opcjonalne?

Reply:
Ustawiłem to opcjonalne bo są jeszcze funkcje do ustawienia tego a na IOS nie jest dostępny ten rootView od razu w komponencie